src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -26,9 +26,21 @@
     var defaultPlanName = 'PLAN-60K';
     var planItems;
     vm.langUsed = $translate.use();
+    vm.showPricingChart = showPricingChart;
+    vm.pricingChartDisplayed = false;
+    vm.planInfoLoader = true;
 
     function activate() {
-      return settings.getPlansAvailable().then(function(response){
```

Comment:
The return value is still returning a promise that does not represent all activation work.